assets/user-profile/components/profile/UserProfile.tsx

```diff
@@ -202,17 +217,21 @@ class UserProfile extends React.Component<IProps, any> {
     }
 }
 
-const mapStateToProps = (state: any) => ({
+const mapStateToProps = (state: IUserProfileState) => ({
     user: state.editedUser,
     errors: state.errors ?? {},
+    authProviderFeatures: state.authProviderFeatures,
 });
 
 const mapDispatchToProps = (dispatch: any) => ({
     saveUser: () => dispatch(saveUser()),
     fetchUser: (userId: IUser['_id']) => dispatch(fetchUser(userId)),
-    onChange: (event: React.ChangeEvent) => dispatch(editUser(event)),
+    onChange: (updates: IUserProfileUpdates) => dispatch(editUser(updates)),
     setError: (errors: {[key: string]: string}) => dispatch(setError(errors)),
     openEditTopicNotificationsModal: () => dispatch(openEditTopicNotificationsModal()),
 });
 
+type IStateProps = typeof mapStateToProps;
```

Comment:
These 2 types aren't used anywhere. Also, the version of Redux we're using doesn't play nicely with typescript (I have updated it in my multiday events PR to the newest version that does)